src/diffpy/srfit/fitbase/recipeorganizer.py

```diff
@@ -933,6 +933,9 @@ def _formatRestraints(self):
         rlines.sort(key=numstr)
         return rlines
 
+    def _matchPattern(self, s):
```

Comment:
following the `camelCase` convention.

Reply:
Noted. Will implement the pep8 standards and make function names specific so that it documents by itself